rebasehelper/application.py

```diff
@@ -621,7 +622,7 @@ def run_package_checkers(self, results_dir):
 
         for checker_name in self.conf.pkgcomparetool:
             try:
-                results[checker_name] = checkers_runner.run_checker(results_dir, checker_name)
+                results[checker_name] = checkers_runner.run_checker(os.path.join(results_dir, 'checkers'), checker_name)
```

Comment:
This line is a bit too long. It fits into the 120 character limit, but it doesn't fit github diff visualizer. Perhaps we should make the limit a bit more strict, say 100 or 110 characters.